.github/workflows/build-push.yml

```diff
@@ -9,59 +9,53 @@ on:
 
 jobs:
   build-publish-proto-dependencies-focal-fossa:
-    runs-on: ubuntu-latest
     name: Build and publish proto dependencies image on ubuntu 20.04
-    steps:
-      - name: Checkout Repository
-        uses: actions/checkout@v2
-
-      - name: Build and Push
-        uses: ./.github/actions/build-push
-        with:
-          image: "Protocol dependencies"
-          base_tag: 20.04
-          secret: ${{ secrets.GITHUB_TOKEN }}
+    uses: aica-technology/.github/.github/workflows/build-push-multi-arch.yml@ed4ba3f4327b2a61948cde7b4e9410363114c478
+    with:
+      image_name: ${{ github.repository }}/proto-dependencies
+      image_tags: 20.04
+      build_context_path: .
```

Comment:
Would be cleaner if some of these inputs could be left out, as this is certainly the default case

Reply:
Yeah that's true, want me to change it?

Reply:
Nitpick only, it would require a change in the other workflow, so up to you

Reply:
If we tag it, then I will fix it